src/components/blocks/canada/south-asia-explore-grid.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { StaticQuery, graphql } from "gatsby"
+import { Row, Col } from "react-bootstrap";
+import PageContainer from 'components/shared/pageContainer'
+import styled from "styled-components"
+
+const yaml = require('js-yaml');
+
+const Background = styled(PageContainer.FullWidth)`
+  background: #F4F7FA;
+`
+const HeadingIcon = styled.i`
+  color: ${props => (props.iconColour ?? "#000000")};
+`
```

Comment:
Is it possible to do this without `styled-components` ? For example, create a class with the background color, put it in `global.css` and then apply that to the PageContainer. Same could be done for the icon colors using the `classNames` utility, e.g.

`<i aria-hidden={true} className={classNames("pe-3",icon_color)} />`

...where `icon_color` is a class declared in `global.css`

Reply:
Yes, but it's too big a change to do before tomorrow.

Instead, I think we should add a story to the backlog to convert all of the explore grid Yaml blocks to an actual widget (it looks like it's being used multiple times so it might be worth converting it). And when we create the widget, we can phase out styled components.